### DISCUSSION TITLE
Unable to Setup Budibase

### DISCUSSION THREAD

**Discussion by starmomou** [Budibase/budibase#3213] [Help] [answered]
Hi, I'm using cmder to run the docker command for self-hosting. I followed the self-hosting options and instructions on budibase website. However, there's something wrong when I moved to Step 2 

```
1. Install the Budibase CLI:

npm i -g @budibase/cli

2. Setup Budibase (select where to store Budibase, and the port to run it on):

budi hosting --init
```

/****************************My Step****************************/
```
C:\Users\lily.mou
λ npm i -g @budibase/cli
C:\Users\lily.mou\AppData\Roaming\npm\budi -> C:\Users\lily.mou\AppData\Roaming\npm\node_modules\@budibase\cli\src\index.js
+ @budibase/cli@0.9.173
added 208 packages from 213 contributors in 31.293s

C:\Users\lily.mou
λ budi hosting --init
? This will create multiple files in current directory, should continue? Yes
Error - Error: Request failed with status code 502
```

Can someone please tell me how to troubleshoot this 502 error message? Thank you

**Comment by mjashanks** [comment, marked as answer]
Hi @starmomou - interesting, the only thing going on at this point is files being downloaded from GitHub.

Can you browse to this URL: https://raw.githubusercontent.com/Budibase/budibase/master/hosting/docker-compose.yaml ?

If so, it's worth trying again in case it was an intermittent issue with GitHub.

**Reply by starmomou** [reply]
@mjashanks No wonder! I forgot to download the docker-compose.yaml files to my host. Now the problem has been solved. Thank you!!
